### PR TITLE
Update permissions-filtering-for-content-search.md

### PR DESCRIPTION
**Pull request by shelri**
This is a part of a larger update of documentation for eDiscovery cases we have seen. We have a 4,000 character limit for sharepoint and ODB site searches that customers are hitting due to how searches are expanded on the backend.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/permissions-filtering-for-content-search.md [hunk 1]
```diff
@@ -262,3 +262,5 @@ The **Remove-ComplianceSecurityFilter** is used to delete a search filter. Use t
...
     
 - **Does allowing a user to search all content locations in a specific service also prevent them from searching content locations in a different service?** No. As previously explained, you have to create a search permissions filter to explicitly prevent users from searching content locations in a specific service (such as preventing a user from searching any Exchange mailbox or any SharePoint site). In other words, creating a search permissions filter that allows a user to search all SharePoint sites in the organization doesn't prevent that user from searching mailboxes. For example, to allow SharePoint admins to only search SharePoint sites, you have to create a filter that prevents them from searching mailboxes. Similarly, to allow Exchange admins to only search mailboxes, you have to create a filter that prevents them from searching sites.
+
+- **Do Compliance Security Filters Count against Search Query Limits?** Yes. Compliance Security filters do count against the character limit for Search queries. For more information, see [Limits in Advanced eDiscovery](https://docs.microsoft.com/en-us/microsoft-365/compliance/limits-ediscovery20?view=o365-worldwide).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/permissions-filtering-for-content-search.md]:
```suggestion
- **Do compliance security filters count against search query limits?** Yes. Compliance security filters do count against the character limit for search queries. For more information, see [Limits in Advanced eDiscovery](https://docs.microsoft.com/microsoft-365/compliance/limits-ediscovery20?view=o365-worldwide).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @shelri please commit my suggestions.

**Comment by yogkumgit** [comment]
Gentle reminder @shelri Please update with feedback

**Comment by markjjo** [comment]
@yogkumgit I am working offline with @shelri and the PM on this PR. It's assigned to me and I will take ownership of any additional changes that need to be made.

**Comment by markjjo** [comment]
@shelri PR merged. Sorry for the delay.

**Pull request merged**